schema-source/RSxxSub.schema.yaml

```diff
@@ -0,0 +1,224 @@
+Schema:
+  Object Type: "Meta"
+  References:
+    - "ASHRAE205"
+  Title: "Fenestration - Glazing Substrate"
+  Description: "Schema for ASHRAE 205 annex RSxxSub: Fenestration - Glazing Substrate" 
+  Version: "2.0.0"
+  Root Data Group: "RSxxSub"
+
+# Data Groups
+RSxxSub:
+  Object Type: "Data Group"
+  Data Elements:
+    metadata:
+      Description: "Metadata data group"
+      Data Type: "{Metadata}"
+      Constraints: "schema=RSxxSub"
+      Required: True
+    description:
+      Description: "Data group describing product and rating information"
+      Data Type: "{Description}"
+    performance:
+      Description: "Data group containing performance information"
+      Data Type: "{Performance}"
+      Required: True
+
+Description:
+  Object Type: "Data Group"
+  Data Elements:
+    product_information:
+      Description: "Data group describing product information"
+      Data Type: "{ProductInformation}"
+
+ProductInformation:
+  Object Type: "Data Group"
+  Data Elements:
+    manufacturer:
+      Description: "Manufacturer name"
+      Data Type: "String"
+    model_number:
+      Description: "Model number"
+      Data Type: "Pattern"
+      Notes: "Pattern shall match all model numbers that can be represented by the representation"
+
+Performance:
+  Object Type: "Data Group"
+  Data Elements:
+    thickness:
+      Description: "The total thickness of the glazing."
+      Data Type: "Numeric"
+      Units: "m"
+      Required: True
+    thermal_conductivity:
+      Description: "Overall thermal conductivity of the glazing."
+      Data Type: "Numeric"
+      Units: "W/m-K"
+      Required: True
+    front_side_infrared_emissivity:
+      Description: "Front Side Infrared Emissivity"
+      Data Type: "Numeric"
+      Units: "-"
+      Required: True
+      Constraints: [">=0.0", "<=1.0"]
+    back_side_infrared_emissivity:
+      Description: "Back Side Infrared Emissivity"
+      Data Type: "Numeric"
+      Units: "-"
+      Required: False
+      Constraints: [">=0.0", "<=1.0"]
+      Notes: "When this data element is missing, the value is assumed to be the same as the front_side_infrared_emissivity."
+    is_coated:
+      Description: "Indicates that the front side of the glass has a coating or applied film"
+      Data Type: "Boolean"
+      Required: True
+      Notes: "The properties inculde the effect of the coating or film."
+    performance_map_optical:
+      Description: "Data group describing optical performance over a range of conditions. A list is provided that corresponds to the lists of wavelength_integration, is_incidence_hemispherical, and is_emergence_hemispherical."
+      Data Type: "[{PerformanceMapOptical}]"
```

Comment:
This array will in general contain 6 types of objects because the irradiance and the emergence can be provided as a string like "hemispherical" or bidirectional with two angles and the wavelengths can be integrated or spectrally resolved. This means a software developer who shall use the data needs to understand the differences between between the 6 types, implement them and not confuse them. Why don't exchange the the optical data as BED-JSON which is easy to understand and to parse?

Reply:
In general we are trying to stay consistent with the GridVariable and LookUp variable approach that Standard 205 is using for HVAC data. I was trying to keep this simple. I'm not sure what you are saying is confusing, could you please elaborate?